tests/models/test_playerscore.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from mars_api.models import Game, Player, PlayerScore
+
+pytestmark = pytest.mark.django_db
+
+
+def test_unique_players_for_the_same_game():
+    """
+    Tests if only unique players can be added to the PlayerScore with same game_id.
+    (So that there wouldn't be a player that has two scores for the same Game.)
```

Comment:
Should this be part of `playerscore` or `game` tests?

Reply:
The error will be thrown when creating a new `PlayerScore` object, thus I thought the validation logic would be done in `PlayerScore` model.

Reply:
look into `unique_together`

Reply:
Good point, though I will be using `unique_constraint` because of:
![image](https://user-images.githubusercontent.com/34490649/89736455-69353900-da61-11ea-8ce5-40d2b5603474.png)

Reply:
noice